apiary.apib

```diff
@@ -6733,6 +6736,50 @@ For a more detailed guide on the file upload, see the corresponding part of the
                 }
             }
 
++ Response 202 (application/json)
+    Asynchronous call response (async=true) that creates a new job. To see job detail use [Job Detail API endpoint](#reference/jobs/manage-jobs/job-detail)
+    + Body
+
+           {
+               "id": 11,
+               "status": "waiting",
+               "url": "https://connection.keboola.com/v2/storage/jobs/11",
+               "operationName": "workspaceCreate",
+               "operationParams": {
+                   "name": "filePrepare",
+                   "isPermanent": true,
+                   "isSliced": false,
+                   "contentType": "text/csv",
+                   "tags": [
+                       "tag1",
+                       "tag2"
+                   ],
+                   "sizeBytes": 123456,
+                   "isPublic": true,
```

Comment:
```suggestion
                   "isPublic": false,
``` 
je to deprecated

Reply:
to je ukázka response to je docela fuk ale upravil sem